great_expectations/data_context/data_context/base_data_context.py

```diff
@@ -870,7 +608,8 @@ def datasources(self) -> Dict[str, Union[LegacyDatasource, BaseDatasource]]:
     @property
     def checkpoint_store_name(self):
         try:
-            return self.project_config_with_variables_substituted.checkpoint_store_name
+            return super().checkpoint_store_name
+
```

Comment:
Is this necessary to include? `checkpoint_store_name` should already be inherited via EphemeralDataContext -> AbstractDataContext.